pkg/packetfilter/packetfilter.go

```diff
@@ -272,6 +272,10 @@ type NamedSet interface {
 }
 
 type Driver interface {
+	// MSS Clamp rules should be configured in different tables for iptables and nftables,
+	// TableTypeRoute for iptables and TableTypeFilter for nftables
+	GetMSSClampTable() TableType
```

Comment:
Why don't we define a new `TableType` enum named "_MSSClamp_" instead of adding a new interface method? Then the implementation can internally decide what it should translate to.

Reply:
Also thought to go with this approach first, for this approach we need two new values : 
1. new table type e.g: **TableTypeMssClamp**  , 
    and **AppendUnique , ClearChain , UpdateChainRules** APIs implementation should be updated to support this value 

2. new chain type e.g: **ChainTypeMssClamp**
 and **CreateIPHookChainIfNotExists , DeleteIPHookChain**  APIs implementation should be updated to support this value 

But in second thought, adding new GetMSSClampTable() API seems like a simpler solution to me